assignments/objects.js

```diff
@@ -18,28 +18,77 @@ const example = {
 }
 
 // Write your intern objects here:
+function Intern(i_id, i_email, i_name, i_gender) { 
```

Comment:
This is clever - but it wanted you to follow the example on line 13, and create individual objects for each intern (basically practice writing object format)